src/html/partials/key-factors.html

```diff
@@ -1,4 +1,40 @@
 <section class="factors-section">
+  
   <h2 class="sm-title">Why People Don't Adhere</h2>
   <p class="bg-paragraph">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
+
+  <div class="factors-section__human-figure-wrapper">
+    <div class="factors-section__human-figure-gear"></div>
+    <div class="factors-section__human-figure-heart"></div>
+    <div class="factors-section__human-figure-dollar"></div>
+    <div class="factors-section__human-figure-tally"></div>
+  </div>
+
+  <ul class="factors-section__list">
+    <li class="factors-section__list__item">
+      <div class="factors-section__gear"></div>
+      <h3>Comprehension</h3>
+      <p>“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly
+        questions.”</p>
+    </li>
+
+    <li class="factors-section__list__item">
+      <div class="factors-section__heart"></div>
+      <h3>Denial</h3>
+      <p>“I feel just fine. I don’t know why that doctor thinks I’m sick. Why would I take medicine that I don’t need?”</p>
+    </li>
+
+    <li class="factors-section__list__item">
+      <div class="factors-section__dollar"></div>
+      <h3>Cost</h3>
+      <p>“I took my prescription to the pharmacy but they said it would cost several hundred dollars! I just couldn’t afford it.”</p>
+    </li>
+
+    <li class="factors-section__list__item">
+      <div class="factors-section__tally"></div>
```

Comment:
Similar thought here with these empty divs. You maaaay not need these particular ones once you get to the styling part, but go ahead and leave them in for now.